cluster/HDdistances.py

```diff
@@ -0,0 +1,71 @@
+
+# This file provides functionalities for High dimensionality problems but also for low dimensionality problems
+#   - New Distance computation
+#   - SSE metric computation for assist the computation of the optimal number of clusters
+#
+# Authors:
+# Jose Javier Garcia Aranda
+# Juan Ramos Diaz
+
+
+
+#from cluster import KMeansClustering
+#import KMeansClustering
+#import ClusteringError
+import util
+import time
+import datetime
+
+import random
+
+def HDdistItems(profile1,profile2):
+    #Distance function, this distance between two profiles is based on:
+    #For each keyword of user A, if the keyword is not  present in user B , then the distance for this keyword is the weight in the user A.
+    #If the keyword exists in both users, the weights are compared and the distance is the absolute difference
+    len1=len(profile1)/2
```

Comment:
Has this been tested in both Python 2 & 3? Division behaves a bit differently in both 2 & 3. In Python 2 this will always return an `int` (rounding down), whereas in Python 3 this might return a `float` on uneven lengths!